### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by nathanq95**
Created a seeder that populates the database with all sizes & styles, each size having a random quantity.  Started work on server and database index files.

### CODE REVIEW

File: seed/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+
+
+let addSizesToDB = (productId, styleId) => {
+    let newSizes = sizes.generateSizes(productId, styleId);
+    
+    for (let currentSize = 0; currentSize < newSizes.length; currentSize++) {
+        
+        inventoryDB.addToDb(newSizes[currentSize])
```
Comment (anabellchan) [inline seed/index.js]: If you have 100 products X 3 styles, round trip from DB is 300 times.  To be performant, you can put all your objects in an array first, then insert that into DB.

File: seed/generateSizes.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+
+const menSizes = [6, 6.5, 7, 7.5, 8, 8.5, 9, 9.5, 10, 10.5, 11, 11.5, 12, 12.5, 13, 14, 15];
+const womenSizes = [5, 5.5, 6, 6.5, 7, 7.5, 8, 8.5, 9, 9.5, 10, 10.5, 11, 11.5, 12];
+
+let generateRandomQuantity = () => {
+    let quantity = Math.floor(Math.random() * Math.floor(10));
```
Comment (anabellchan) [inline seed/generateSizes.js:6]: Did you mean 
`Math.floor(Math.random() * 10);`

**Comment by anabellchan** [comment]
Awesome work Nathan!  Make sure you create a Trello card to add unit test for your database calls.

**Review by anabellchan — APPROVED** [review]

File: seed/index.js (hunk 1 above)
Comment (nathanq95) [reply seed/index.js]: Do you mean adding the actual array to the database?  I thought the agreed upon structure was one document per size, so for 300 sizes I'd need to query the db for every size

File: seed/generateSizes.js (hunk 2 above)
Comment (nathanq95) [reply seed/generateSizes.js:6]: I can change it to that.  I just followed an example I found in MDN

**Pull request merged**